### PR TITLE
Add option: auto-play movie on window maximize

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Changelog {#changelog}
 ============
+# Release 1.5.2 (git master)
+
+* [298](https://github.com/BlueBrain/Tide/pull/298):
+  New feature in options: a stopped movie can be auto-played when maximized.
```
Comment (karjonas) [inline doc/Changelog.md]:
```suggestion
  New option: start/resume movie playback when window is maximized.
```

File: tide/master/control/DisplayGroupController.h [hunk 2]
```diff
@@ -42,6 +42,7 @@
 
 #include "types.h"
 
+#include "scene/Options.h"
```
Comment (karjonas) [inline tide/master/control/DisplayGroupController.h]:
```suggestion
```

**Comment by karjonas** [comment]
Maybe rename PR to something like "Add option: auto-play movie on window maximize"?

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Pull request merged**